src/powershell/Public/Get-FinOpsCostExport.ps1

```diff
@@ -0,0 +1,176 @@
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Get list of Cost Management Exports.
+
+    .PARAMETER Name
+    Optional. Name of the export. Supports wildcards.
+
+    .PARAMETER Scope
+    Optional. Resource ID of the scope the export was created for. If empty, defaults to current subscription context.
+
+    .PARAMETER DataSet
+    Optional. Dataset to get exports for. Allowed values = "ActualCost", "AmortizedCost", "Usage". Default = null (all exports).
+
+    .PARAMETER StorageAccountId
+    Optional. Resource ID of the storage account to get exports for. Default = null (all exports).
+
+    .PARAMETER StorageContainer
+    Optional. Name of the container to get exports for. Supports wildcards. Default = null (all exports).
+
+    .PARAMETER ApiVersion
+    Optional. API version to use when calling the Cost Management Exports API. Default = 2023-03-01.
+
+    .EXAMPLE
+    Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000"
+
+    Gets all exports for a subscription. Does not include exports in nested resource groups.
+    
+    .EXAMPLE
+    Get-FinOpsCostExport -Name mtd* -Scope "providers/Microsoft.Billing/billingAccounts/79676095"
+
+    Gets export with name matching wildcard mtd* within the specified billing account scope. Does not include exports in nested resource groups.
+
+    .EXAMPLE
+    Get-FinOpsCostExport -DataSet "AmortizedCost"
+
+    Gets all exports within the current context subscription scope and filtered by dataset AmortizedCost.
+
+    .EXAMPLE
+    Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -StorageAccountId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Storage/storageAccounts/MyStorageAccount"
+
+    Gets all exports within the subscription scope filtered by a specific storage account.
+
+    .EXAMPLE
+    Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -StorageContainer "MyContainer*"
+
+    Gets all exports within the subscription scope for a specific container. Supports wildcard.
+
+    .EXAMPLE
+    Get-FinOpsCostExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -StorageContainer "mtd*" -apiVersion "2023-08-01"
+
+    Gets all exports within the subscription scope for a container matching wildcard pattern and using a specific API version.
+#>
+function Get-FinOpsCostExport
+{
+    param
+    (
+        [Parameter()]
+        [SupportsWildcards()]
+        [string]
+        $Name,
+
+        [Parameter()]
+        [string]
+        $Scope,
+
+        [Parameter()]
+        [ValidateSet("ActualCost", "AmortizedCost", "Usage")]
+        [string]
+        $DataSet,
+
+        [Parameter()]
+        [string]
+        $StorageAccountId,
+
+        [Parameter()]
+        [SupportsWildcards()]
+        [string]
+        $StorageContainer,
+
+        [Parameter()]
+        [string]
+        $APIVersion = '2023-08-01'
```

Comment:
nit: Microsoft's standard is to only capitalize the first character of acronyms with 3+ letters.
```suggestion
        $ApiVersion = '2023-08-01'
```